api/unit-test.sh

```diff
@@ -3,16 +3,18 @@
 export NODE_ENV=test
 
 docker-compose -f ../docker-compose.endpoint-tests.yml -p api up -d
+sleep 5
```

Comment:
why does this need to be added?

Reply:
I don't know that it _has_ to be added, but sometimes it seems like the servers aren't fully up before it moves to the next step, so it's kind of insurance